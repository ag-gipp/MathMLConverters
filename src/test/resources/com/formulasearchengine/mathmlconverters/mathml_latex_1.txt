<formula xmlns="http://www.tei-c.org/ns/1.0" rend="display">W(t) = Wmax [1 - exp(- b t)]</formula>