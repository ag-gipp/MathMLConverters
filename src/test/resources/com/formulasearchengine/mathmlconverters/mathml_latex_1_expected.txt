<math xmlns="http://www.w3.org/1998/Math/MathML" id="p1.1.m1.1" alttext="W(t)=Wmax[1-exp(bt)]">
  <semantics id="p1.1.m1.1a">
    <mrow id="p1.1.m1.1.22" xref="p1.1.m1.1.22.cmml">

        <mi id="p1.1.m1.1.1" xref="p1.1.m1.1.1.cmml">W</mi>


          <mrow><mo id="p1.1.m1.1.22.1.2">(</mo><mi id="p1.1.m1.1.3" xref="p1.1.m1.1.3.cmml">t</mi><mo id="p1.1.m1.1.22.1.2a">)</mo></mrow>




      <mo id="p1.1.m1.1.5" xref="p1.1.m1.1.5.cmml">=</mo>

        <mi id="p1.1.m1.1.6" xref="p1.1.m1.1.6.cmml">W</mi>

        <mi id="p1.1.m1.1.7" xref="p1.1.m1.1.7.cmml">m</mi>

        <mi id="p1.1.m1.1.8" xref="p1.1.m1.1.8.cmml">a</mi>

        <mi id="p1.1.m1.1.9" xref="p1.1.m1.1.9.cmml">x</mi>


          <mrow><mo id="p1.1.m1.1.22.2.2.1.1">[</mo><mrow><mn id="p1.1.m1.1.11" xref="p1.1.m1.1.11.cmml">1</mn><mo id="p1.1.m1.1.12" xref="p1.1.m1.1.12.cmml">-</mo><mi id="p1.1.m1.1.13" xref="p1.1.m1.1.13.cmml">e</mi><mi id="p1.1.m1.1.14" xref="p1.1.m1.1.14.cmml">x</mi><mi id="p1.1.m1.1.15" xref="p1.1.m1.1.15.cmml">p</mi><mrow><mo id="XM3.1.2">(</mo><mrow><mo id="p1.1.m1.1.17" xref="p1.1.m1.1.17.cmml">-</mo><mi id="p1.1.m1.1.18" xref="p1.1.m1.1.18.cmml">b</mi><mi id="p1.1.m1.1.19" xref="p1.1.m1.1.19.cmml">t</mi></mrow><mo id="XM3.1.2a">)</mo></mrow></mrow><mo id="p1.1.m1.1.22.2.2.1.1a">]</mo></mrow>



























    </mrow>
    <annotation-xml encoding="MathML-Content" id="p1.1.m1.1b">
      <apply id="p1.1.m1.1.22.cmml" xref="p1.1.m1.1.22">
        <eq id="p1.1.m1.1.5.cmml" xref="p1.1.m1.1.5" />
        <apply id="p1.1.m1.1.22.1.cmml" xref="p1.1.m1.1.22.1">
          <times id="p1.1.m1.1.22.1.1.cmml" xref="p1.1.m1.1.22.1.1" />
          <ci id="p1.1.m1.1.1.cmml" xref="p1.1.m1.1.1">W</ci>
          <ci id="p1.1.m1.1.3.cmml" xref="p1.1.m1.1.3">t</ci>
        </apply>
        <apply id="p1.1.m1.1.22.2.cmml" xref="p1.1.m1.1.22.2">
          <times id="p1.1.m1.1.22.2.1.cmml" xref="p1.1.m1.1.22.2.1" />
          <ci id="p1.1.m1.1.6.cmml" xref="p1.1.m1.1.6">W</ci>
          <ci id="p1.1.m1.1.7.cmml" xref="p1.1.m1.1.7">m</ci>
          <ci id="p1.1.m1.1.8.cmml" xref="p1.1.m1.1.8">a</ci>
          <ci id="p1.1.m1.1.9.cmml" xref="p1.1.m1.1.9">x</ci>
          <apply id="p1.1.m1.1.22.2.2a.cmml">
            <csymbol id="p1.1.m1.1.22.2.2.cmml">delimited-[]</csymbol>
            <apply id="XM3.cmml" xref="XM3">
              <minus id="p1.1.m1.1.12.cmml" xref="p1.1.m1.1.12" />
              <cn type="integer" id="p1.1.m1.1.11.cmml" xref="p1.1.m1.1.11">1</cn>
              <apply id="XM3.1.cmml" xref="XM3.1">
                <times id="XM3.1.1.cmml" xref="XM3.1.1" />
                <ci id="p1.1.m1.1.13.cmml" xref="p1.1.m1.1.13">e</ci>
                <ci id="p1.1.m1.1.14.cmml" xref="p1.1.m1.1.14">x</ci>
                <ci id="p1.1.m1.1.15.cmml" xref="p1.1.m1.1.15">p</ci>
                <apply id="XM2.cmml" xref="XM2">
                  <minus id="p1.1.m1.1.17.cmml" xref="p1.1.m1.1.17" />
                  <apply id="XM2.1.cmml" xref="XM2.1">
                    <times id="XM2.1.1.cmml" xref="XM2.1.1" />
                    <ci id="p1.1.m1.1.18.cmml" xref="p1.1.m1.1.18">b</ci>
                    <ci id="p1.1.m1.1.19.cmml" xref="p1.1.m1.1.19">t</ci>
                  </apply>
                </apply>
              </apply>
            </apply>
          </apply>
        </apply>
      </apply>
    </annotation-xml>
    <annotation encoding="application/x-tex" id="p1.1.m1.1c">W(t)=Wmax[1-exp(bt)]</annotation>
  </semantics>
</math>
